css/properties.json

```diff
@@ -703,24 +707,24 @@
     "groups": [
       "WebKit Extensions"
     ],
-    "initial": "<code>rgb(0, 0, 0)</code>",
+    "initial": "<code>currentColor</code>",
     "appliesto": "allElements",
-    "computed": "asSpecified",
+    "computed": "'color'",
     "order": "uniqueOrder",
     "status": "nonstandard"
   },
   "-webkit-text-stroke-width": {
     "syntax": "&lt;length&gt;",
     "media": "visual",
     "inherited": true,
-    "animationType": "discrete",
+    "animationType": "no",
     "percentages": "no",
     "groups": [
       "WebKit Extensions"
     ],
     "initial": "<code>0</code>",
     "appliesto": "allElements",
-    "computed": "asSpecified",
+    "computed": "absolute width",
```

Comment:
This should be written as "absoluteWidth" (as l10n keyword) and get translated in [l10n/css.json](https://github.com/mdn/data/blob/master/l10n/css.json).

Sebastian